aten/src/ATen/native/LossNLL2d.cpp

```diff
@@ -499,13 +499,4 @@ Tensor nll_loss2d_symint(const Tensor & self, const Tensor & target, const std::
   return std::get<0>(at::nll_loss2d_forward_symint(self, target, weight, reduction, std::move(ignore_index)));
 }
 
-// Duplicate of above code for non-symbolic ints. Kept for BC purposes and to minimize breakages.
```

Comment:
This says kept for BC purposes, will this really not break anything?

Reply:
This file is rarely changed. Maybe the comments were added long time ago.